Fuse.pretty/Fuseholder_Clip-5x20mm_Eaton_1A5601-01_Inline_P20.80x6.76mm_D1.70mm_Horizontal.kicad_mod

```diff
@@ -0,0 +1,50 @@
+  (fp_text value Fuseholder_Clip-5x20mm_Eaton_1A5601-01_Inline_P20.80x6.76mm_D1.70mm_Horizontal (at 10.4 4.5) (layer F.Fab)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_line (start 6.53 -2.5) (end 14.27 -2.5) (layer F.Fab) (width 0.1))
+  (fp_line (start 14.27 2.5) (end 6.53 2.5) (layer F.Fab) (width 0.1))
+  (fp_line (start 14.27 -3.175) (end 14.27 3.175) (layer F.Fab) (width 0.1))
+  (fp_line (start 14.27 3.175) (end 20.57 3.175) (layer F.Fab) (width 0.1))
+  (fp_line (start 20.57 3.175) (end 20.57 -3.175) (layer F.Fab) (width 0.1))
+  (fp_line (start 20.57 -3.175) (end 14.27 -3.175) (layer F.Fab) (width 0.1))
+  (fp_line (start 0.23 -3.175) (end 6.53 -3.175) (layer F.Fab) (width 0.1))
+  (fp_line (start 6.53 -3.175) (end 6.53 3.175) (layer F.Fab) (width 0.1))
+  (fp_line (start 6.53 3.175) (end 0.23 3.175) (layer F.Fab) (width 0.1))
+  (fp_line (start 0.23 3.175) (end 0.23 -3.175) (layer F.Fab) (width 0.1))
+  (fp_line (start 14.27 -3.175) (end 14.27 -1.6) (layer F.SilkS) (width 0.12))
+  (fp_line (start 14.27 3.175) (end 20.57 3.175) (layer F.SilkS) (width 0.12))
+  (fp_line (start 20.57 -3.175) (end 14.27 -3.175) (layer F.SilkS) (width 0.12))
+  (fp_line (start 14.27 2.5) (end 6.53 2.5) (layer F.SilkS) (width 0.12))
+  (fp_line (start 14.27 -2.5) (end 6.53 -2.5) (layer F.SilkS) (width 0.12))
+  (fp_line (start 20.57 1.6) (end 20.57 3.175) (layer F.SilkS) (width 0.12))
+  (fp_line (start 6.53 -3.175) (end 0.23 -3.175) (layer F.SilkS) (width 0.12))
+  (fp_line (start 0.23 -3.175) (end 0.23 -1.6) (layer F.SilkS) (width 0.12))
+  (fp_line (start 6.53 3.175) (end 0.23 3.175) (layer F.SilkS) (width 0.12))
+  (fp_line (start 6.53 -3.175) (end 6.53 -1.6) (layer F.SilkS) (width 0.12))
+  (fp_line (start -1.6 -3.45) (end 22.4 -3.45) (layer F.CrtYd) (width 0.05))
+  (fp_line (start -1.6 -3.45) (end -1.6 3.45) (layer F.CrtYd) (width 0.05))
+  (fp_line (start 22.4 3.45) (end 22.4 -3.45) (layer F.CrtYd) (width 0.05))
+  (fp_line (start 22.4 3.45) (end -1.6 3.45) (layer F.CrtYd) (width 0.05))
+  (fp_text user %R (at 10.4 0) (layer F.Fab)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_line (start 0.23 3.175) (end 0.23 1.6) (layer F.SilkS) (width 0.12))
+  (fp_line (start 20.57 -1.6) (end 20.57 -3.175) (layer F.SilkS) (width 0.12))
+  (fp_line (start 6.53 3.175) (end 6.53 1.6) (layer F.SilkS) (width 0.12))
+  (fp_line (start 14.27 3.175) (end 14.27 1.6) (layer F.SilkS) (width 0.12))
+  (pad 2 thru_hole circle (at 14.04 0) (size 2.7 2.7) (drill 1.7) (layers *.Cu *.Mask))
+  (pad 2 thru_hole circle (at 20.8 0) (size 2.7 2.7) (drill 1.7) (layers *.Cu *.Mask))
+  (pad 1 thru_hole circle (at 6.76 0) (size 2.7 2.7) (drill 1.7) (layers *.Cu *.Mask))
+  (pad 1 thru_hole circle (at 0 0) (size 2.7 2.7) (drill 1.7) (layers *.Cu *.Mask))
+  (model ${KISYS3DMOD}/Fuse.3dshapes/Fuseholder_Clip-5x20mm_Eaton_1A5601-01_Inline_P20.80x6.76mm_D1.70mm_Horizontal.wrl
+    (at (xyz 0 0 0))
+    (scale (xyz 1 1 1))
+    (rotate (xyz 0 0 0))
+  )
+)
```

Comment:
Also i think pitch should be greater than 20.8mm in order to fit a 20x5 fuse.
![εικόνα](https://user-images.githubusercontent.com/48727784/94375249-0ab04100-011b-11eb-8b45-b8d9cde372bd.png)
![εικόνα](https://user-images.githubusercontent.com/48727784/94375278-40edc080-011b-11eb-84e7-9251a2ea96d7.png)

Reply:
The outer pads are 20.8mm apart according to the datasheet. Last page.
![image](https://user-images.githubusercontent.com/1936989/94517495-e5840700-01dc-11eb-8aef-e768719a83ec.png)

I think I mis-read the datasheet on the width. Fixed now. Thank you.